### PR TITLE
Add an option to display relative line numbers

### LINKED ISSUE #265: Feature request: Show relative line numbers
Please add the option to show line numbers relative to the current/active line.

```
3 | ### My fancy Header 3
2 | This is my lovely paragraph...
1 |
0 | * This line is the line I'm currently editing (it's the first item of a markdown list)
1 | * another item
2 | * another item
3 |
```

If both line numbers and relative line numbers are turned on, something like vim's hybrid line numbers would be simply superb! See: http://jeffkreeftmeijer.com/2013/vims-new-hybrid-line-number-mode/

hybridnumber

### PR DESCRIPTION
**Pull request by xgouchet**
With this new option, the line numbers are counted as relative from the
current line.

Fixes #265

### CODE REVIEW

File: ReText/__init__.py [hunk 1]
```diff
@@ -96,7 +97,6 @@ def getBundledIcon(iconName):
 	'useWebKit': False,
 	'windowGeometry': QByteArray(),
 }
-
```
Comment (mitya57) [inline ReText/__init__.py]: Please do not remove whitespace here.

File: ReText/editor.py [hunk 2]
```diff
@@ -350,11 +352,15 @@ def paintEvent(self, event):
...
+		if globalSettings.relativeLineNumbers:
+			relativeTo = self.editor.textCursor().blockNumber()
+		else:
+			relativeTo = -1
 		while not atEnd:
 			rect = self.editor.cursorRect(cursor)
 			block = cursor.block()
 			if block.isVisible():
-				number = str(cursor.blockNumber() + 1)
+				number = str(cursor.blockNumber() - relativeTo)
```
Comment (mitya57) [inline ReText/editor.py]: Can you maybe use a proper typographic minus (&minus;) here? I.e. add `.replace('-', '−')` to this string?

Comment (xgouchet) [inline ReText/editor.py]: Sure

**Comment by mitya57** [comment]
Merged, thanks!

**Pull request merged**